tools/fuzzers/fuzzer-json/fuzzer-json.cpp

```diff
@@ -0,0 +1,102 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+#include <hermes/hermes.h>
+#include <jsi/jsi.h>
+#include "hermes/Parser/JSONParser.h"
+#include "libprotobuf-mutator/src/libfuzzer/libfuzzer_macro.h"
```

Comment:
Place a newline between logical groups of includes.
Groups should be ordered as follows:

* Hermes headers
* Third party headers
* Standard library headers

Like so:
```
#include "hermes/hermes.h"
#include "jsi/jsi.h"

#include "libprotobuf-mutator/src/libfuzzer/libfuzzer_macro.h"
#include "json.pb.h"

#include <sstream>
#include <string>
```